mod/deckorate_datasets/set/type/dataset.rb

```diff
@@ -1,6 +1,7 @@
 include_set Abstract::Thumbnail
 include_set Abstract::DeckorateTabbed
 include_set Abstract::Bookmarkable
+include_set Abstract::Breadcrumbs
```

Comment:
`Abstract::DeckorateTabbed` includes `Abstract::Header`, which includes `Abstract::Breadcrumbs`. So this is fine but not really necessary.